File: FizzBuzz.py
def FizzBuzz(lowerBound, upperBound, FirstPattern, SecondPattern):
    for i in range(lowerBound, upperBound):
        if ((i % FirstPattern == 0) and (i % SecondPattern == 0)):
            print("FizzBuzz")
        elif (i % FirstPattern == 0):
            print("Fizz")
        elif (i % SecondPattern == 0):
            print("Buzz")
        else:
            print(i)


FizzBuzz(1, 16,3,5)

File: FizzBuzz.java
public class FizzBuzz {

    public static void main(String[] args) {
        FizzBuz(1, 100, 4, 6);
    }

    public static void FizzBuz(int lowerBound, int upperBound, int FirstPattern, int SecondPattern) {

        for (int i = lowerBound; i <= upperBound; i++) {
            if ((i % FirstPattern == 0 && i % SecondPattern == 0)) {
                System.out.println("FizzBuzz");
            } else if (i % SecondPattern == 0) {
                System.out.println("Buzz");
            } else if (i % FirstPattern == 0) {
                System.out.println("Fizz");
            } else {
                System.out.println(i);
            }
        }
    }
}
